ManageWikiExtensions.php

```diff
@@ -3044,6 +3044,17 @@
 		],
 		'section' => 'other',
 	],
+	'semanticscribunto' => [
+		'name' => 'Semantic Scribunto',
```

Comment:
This needs to be SemanticScribunto (no space) or it wont load (needs to match exactly with 'name' in extension.json)